lib/fathead/perl_doc/Parsers/parse-faq.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
```

Comment:
The idea being you can build a hash of indexes, their contents and the required parser and process them fairly uniformly [here](https://github.com/duckduckgo/zeroclickinfo-fathead/blob/6506d2a/lib/fathead/perl_doc/Parsers/parse.pl#L173-L199)